styles/wrap-guide.less

```diff
@@ -7,6 +7,15 @@ atom-text-editor::shadow {
     z-index: 3;
     position: absolute;
     top: 0;
-    background-color: @syntax-wrap-guide-color;
+    background-color: transparent;
+
+    &:after {
```

Comment:
Just out of curiosity, is there any reason why this is in a pseudo element instead of in the above class?

(Not that I think there's anything wrong with it per se, I'm just wondering)